### PR TITLE
Update attribute syntax

### LINKED ISSUE #17122: Out of date example for attribute on return?
```fsharp
let function1 x : [<return: Obsolete>] int = x + 1
```

gives me `This attribute is not valid for use on this language element`  on F# 4.7 + dotnet core 3.1

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: aace421d-61eb-5fc0-ec1b-41f3848625f5
* Version Independent ID: e237e964-3f9d-ceb9-8ff6-5cc0813718de
* Content: [Attributes - F#](https://docs.microsoft.com/en-us/dotnet/fsharp/language-reference/attributes)
* Content Source: [docs/fsharp/language-reference/attributes.md](https://github.com/dotnet/docs/blob/master/docs/fsharp/language-reference/attributes.md)
* Product: **dotnet-fsharp**
* GitHub Login: @cartermp
* Microsoft Alias: **phcart**

### PR DESCRIPTION
**Pull request by cartermp**
Fixes https://github.com/dotnet/docs/issues/17122

And updates some of the syntax

### CODE REVIEW

File: docs/fsharp/language-reference/attributes.md [hunk 1]
```diff
@@ -68,19 +68,19 @@ Although you do not usually need to specify the attribute target explicitly, val
   </tr>
   <tr>
     <td>return</td>
-    <td><pre lang="fsharp"><code>let function1 x : [&lt;return: Obsolete&gt;] int = x + 1</code></pre></td>
+    <td><pre lang="fsharp"><code>let function1 x : [&lt;return: MyCustomAttributeThatWorksOnReturns&gt;] int = x + 1</code></pre></td>
```
Comment (mairaw) [inline docs/fsharp/language-reference/attributes.md]:
```suggestion
    <td><pre><code class="lang-fsharp">let function1 x : [&lt;return: MyCustomAttributeThatWorksOnReturns&gt;] int = x + 1</code></pre></td>
```

File: docs/fsharp/language-reference/attributes.md [hunk 2]
```diff
@@ -68,19 +68,19 @@ Although you do not usually need to specify the attribute target explicitly, val
...
   </tr>
   <tr>
     <td>field</td>
-    <td><pre lang="fsharp"><code>[&lt;field: DefaultValue&gt;] val mutable x: int</code></pre></td>
+    <td><pre lang="fsharp"><code>[&lt;DefaultValue&gt;] val mutable x: int</code></pre></td>
```
Comment (mairaw) [inline docs/fsharp/language-reference/attributes.md]:
```suggestion
    <td><pre><code class="lang-fsharp">[&lt;DefaultValue&gt;] val mutable x: int</code></pre></td>
```

File: docs/fsharp/language-reference/attributes.md [hunk 3]
```diff
@@ -68,19 +68,19 @@ Although you do not usually need to specify the attribute target explicitly, val
...
   </tr>
   <tr>
     <td>property</td>
-    <td><pre lang="fsharp"><code>[&lt;property: Obsolete&gt;] this.MyProperty = x</code></pre></td>
+    <td><pre lang="fsharp"><code>[&lt;Obsolete&gt;] this.MyProperty = x</code></pre></td>
```
Comment (mairaw) [inline docs/fsharp/language-reference/attributes.md]:
```suggestion
    <td><pre><code class="lang-fsharp">[&lt;Obsolete&gt;] this.MyProperty = x</code></pre></td>
```

File: docs/fsharp/language-reference/attributes.md [hunk 4]
```diff
@@ -68,19 +68,19 @@ Although you do not usually need to specify the attribute target explicitly, val
...
   </tr>
   <tr>
     <td>param</td>
-    <td><pre lang="fsharp"><code>member this.MyMethod([&lt;param: Out&gt;] x : ref&lt;int&gt;) = x := 10</code></pre></td>
+    <td><pre lang="fsharp"><code>member this.MyMethod([&lt;Out&gt;] x : ref&lt;int&gt;) = x := 10</code></pre></td>
```
Comment (mairaw) [inline docs/fsharp/language-reference/attributes.md]:
```suggestion
    <td><code class="lang-fsharp">member this.MyMethod([&lt;Out&gt;] x : ref&lt;int&gt;) = x := 10</code></pre></td>
```

File: docs/fsharp/language-reference/attributes.md [hunk 5]
```diff
@@ -68,19 +68,19 @@ Although you do not usually need to specify the attribute target explicitly, val
...
   </tr>
   <tr>
     <td>type</td>
```
Comment (mairaw) [inline docs/fsharp/language-reference/attributes.md]: you should do the same change on line 88

**Review by mairaw — APPROVED** [review]
Left you some comments because code formatting was not working.

File: docs/fsharp/language-reference/attributes.md [hunk 6]
```diff
@@ -68,19 +68,19 @@ Although you do not usually need to specify the attribute target explicitly, val
...
   </tr>
   <tr>
     <td>property</td>
-    <td><pre lang="fsharp"><code>[&lt;property: Obsolete&gt;] this.MyProperty = x</code></pre></td>
+    <td><pre><code class="lang-fsharp">[&lt;Obsolete&gt;] this.MyProperty = x</code></pre></td>
   </tr>
   <tr>
     <td>param</td>
-    <td><pre lang="fsharp"><code>member this.MyMethod([&lt;param: Out&gt;] x : ref&lt;int&gt;) = x := 10</code></pre></td>
+    <td><code class="lang-fsharp">member this.MyMethod([&lt;Out&gt;] x : ref&lt;int&gt;) = x := 10</code></pre></td>
```
Comment (mairaw) [inline docs/fsharp/language-reference/attributes.md]: Probably a bad suggestion from me
```suggestion
    <td><pre><code class="lang-fsharp">member this.MyMethod([&lt;Out&gt;] x : ref&lt;int&gt;) = x := 10</code></pre></td>
```

File: docs/fsharp/language-reference/attributes.md [hunk 7]
```diff
@@ -68,19 +68,19 @@ Although you do not usually need to specify the attribute target explicitly, val
   </tr>
```
Comment (mairaw) [inline docs/fsharp/language-reference/attributes.md]: should do a similar change on line 67 as well

Comment (cartermp) [reply docs/fsharp/language-reference/attributes.md]: Which similar change?

**Pull request merged**